src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -131,6 +165,9 @@ export class DictionaryModal extends React.Component {
                       placeholder="English"
                       onChange={this.onChange}
                     >
+                      {isEditingDictionary &&
+                        this.props.preSelectDefaultLocale()
```

Comment:
The `preSelectDefaultLocale` is only returning one `option` element:
```
preSelectDefaultLocale = () => {
   ...
   if (languages[i].value === dictionary.default_locale) {
     return (<option value={languages[i].value} selected>
       {languages[i].label}
     </option>);
  }
  ...
}
```
Are you sure the user would be able to select a new language with this implementation?

Reply:
This implementation  improves  the user experience  by preselecting the option since it is returning existing values for an edit/update form. Therefore the user doesn't have have to change it or any other input element  if that is not what he/she wants to update.

Reply:
Yes, I know that this implementation improves the user experience by preselecting the language option and that is cool 👍.

My question(which i haven't gotten an answer for) was that, **would the user be able to select a new language if he wills to change the language option**?

Reply:
I've made the update